jobs/JREGIONAL_NEXUS_GFS_SFC

```diff
@@ -126,7 +133,7 @@ Call to ex-script corresponding to J-job \"${scrfunc_fn}\" failed."
 #
 #-----------------------------------------------------------------------
 #
-job_postamble "FALSE"
+# job_postamble
```

Comment:
Same as above. Without "FALSE", the KEEPDATA option will not work correctly.

Reply:
This job in one of the four jobs that require DATA to not be removed after complete. The DATA clean up has been handled by a dedicated job data_cleanup.